common/test/acceptance/pages/lms/edxnotes.py

```diff
@@ -523,7 +523,7 @@ def text(self):
         Returns text of the note.
         """
         self.show()
-        element = self.q(css=self._bounded_selector(".annotator-annotation > div"))
+        element = self.q(css=self._bounded_selector(".annotator-annotation > div.annotator-note"))
```

Comment:
Can this just be ``"div.annotator-note"`` now, or does it need to match to the parent too?